examples/quickstart/jupyter-notebooks/sql-tutorial.ipynb

```diff
@@ -0,0 +1,774 @@
+    "- An available Druid instance. This tutorial uses the automatic single-machine configuration described in the [Druid quickstart](https://druid.apache.org/docs/latest/tutorials/index.html), so no authentication or authorization is required unless explicitly mentioned. If you haven’t already, download Druid version 25.0 or higher and start Druid services as described in the quickstart.\n",
+    "\n",
+    "To start the tutorial, run the following cell. It imports the required Python packages and defines a variable for the Druid host, where the Router service listens."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b7f08a52",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "import json\n",
+    "\n",
+    "# druid_host is the hostname and port for your Druid deployment. \n",
+    "# In a distributed environment, you can point to other Druid services.\n",
+    "# In this tutorial, you'll use the Router service as the `druid_host`.\n",
+    "druid_host = \"http://localhost:8888\"\n",
+    "dataSourceName = \"wikipedia-sql-tutorial\"\n",
+    "bold = '\\033[1m'\n",
+    "standard = '\\033[0m'\n",
+    "print(f\"{bold}Druid host{standard}: {druid_host}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e893ef7d-7136-442f-8bd9-31b5a5276518",
+   "metadata": {},
+   "source": [
+    "In the rest of the tutorial, the `endpoint`, `http_method`, and `payload` variables are updated to accomplish different tasks.\n",
+    "\n",
+    "## Druid SQL statements\n",
+    "\n",
+    "The following are the main Druid SQL statements:\n",
+    "\n",
+    "* SELECT: extract data from a datasource\n",
+    "* INSERT INTO: create a new datasource or append to an existing datasource\n",
+    "* REPLACE INTO: create a new datasource or overwrite data in an existing datasource\n",
+    "\n",
+    "Druid SQL does not support CREATE TABLE, DELETE, and DROP TABLE statements.\n",
+    "\n",
+    "## Ingest data\n",
+    "\n",
+    "You can use either INSERT INTO or REPLACE INTO to ingest data into a new datasource.\n",
+    "INSERT INTO and REPLACE INTO statements both require the PARTITIONED BY clause which defines the granularity of time-based partitioning. For more information, see [Partitioning by time](https://druid.apache.org/docs/latest/multi-stage-query/concepts.html#partitioning-by-time).\n",
+    "\n",
+    "Run the following cell to ingest data from an external source into a table named `wikipedia-sql-tutorial` using the [multi-stage query (MSQ) task engine](https://druid.apache.org/docs/latest/multi-stage-query/index.html). The payload is included in the code block directly. If you have an existing datasource with the name `wikipedia-sql-tutorial`, use REPLACE INTO instead of INSERT INTO."
```

Comment:
I think we need to point out a couple things about the query. 1. the time parsing maybe just to say we have a Iso 8601 time column  so wer're parsing it. * says to ingest all the columns in the file. . Then I would mention that the EXTERN statement lets us define the data source type and the input schema. And link out for more information. We don't cover EXTERN anywhere else in this tutorial so want to give folks some way to learn if they can.